src/lib/components/DatePickerPanel.js

```diff
@@ -6,15 +6,28 @@ import classnames from 'classnames';
 import Calendar from 'rc-calendar';
 import TimePickerPanel from 'rc-time-picker/lib/Panel';
 
+// Arbitrary 0.5s interval for live validation of time selection
+const VALIDATION_INTERVAL = 500;
```

Comment:
Given that we don't show the seconds anymore, what do you think about changing this to, like, 10s, or 30s, or even 60s?  🙂